cli/index.js

```diff
@@ -54,17 +53,16 @@ const outputMode = cli.flags.output || 'pretty';
 const outputPath = cli.flags.outputPath || 'stdout';
 const flags = cli.flags;
 
-const driver = new ChromeProtocol();
-
 // set logging preferences
+flags.logLevel = 'info';
```

Comment:
global log but sort of pretending it isn't is still a bit weird, but I don't know a good solution